src/Client/User.php

```diff
@@ -81,4 +81,26 @@ public function deleteUser(string $userId): bool
             throw $this->getExceptionByStatusCode($method, $url, $response);
         }
     }
+
+    public function toggleUser(string $userId, bool $locked): bool
```

Comment:
I don't like the name of this method. Why not `setUserLock()` or something like that? It's not really a toggle if you're gonna pass the value. A toggle is off to on or on to off, not setting a value.

Reply:
Or maybe `setIsLocked()` ?

Reply:
Yeah I basically tried to use what is in the API docs to avoid confusion. But yes we decided to do an opinionated library so it is ok to name things differently I guess. Will change it. 

I will do `setUserLock` as we have `User` in all the other functions already. In hindsight we might should have left that repeating of the ClassName out on functions. Also the classes maybe should have been called `UserClient` and `UserResult` to avoid the need for aliasing classes when both are used. But yeah not too bad either way.

Reply:
These are all good ideas. We can do this, but would need to release a new major version as they are breaking changes. I am for doing this as existing users can just stay on the current version.

Reply:
tracking here https://github.com/btcpayserver/btcpayserver-greenfield-php/issues/100